### ISSUE TITLE
Cardio Risk app incorrectly calculates LDL

### ISSUE THREAD

**Issue by jmandel** [smart-classic/smart_sample_apps#16]
The app is estimating Total Cholesterol - HDL.  this is an overestimate of LDLs, since it doesn't take triglycerides into account.  Should either:
1.  Look for an actual LDL lab result
2.  Look for a triglyceride reading and estimate LDL = Total Cholesterol - HDL - Triglycerides / 5

**Comment by p2** [comment]
The commonly used Friedewald formula for LDL is:

```
LDL = Total Cholesterol - VLDL - HDL
```

_Edit:_ Too fast with submitting. VLDL can be calculated from Triglycerides (if they are below 4 mmol/L / 350 mg/dL):

```
VLDL = Trig / 2.2 [mmol/L]
VLDL = Trig / 5 [mg/dL or g/L]
```

_Edit 2:_ Now I see that that's what you're using, I was used to the 2.2. :D
Nevermind.

**Comment by jmandel** [comment]
No, being explicit about units is important!  The current app is also fragile in that regard...

I just did a quick 'spike' to explore porting the app to HL7's FHIR, and had to add units conversion in the process:
_code_: https://github.com/chb/fhir_cardiac_risk/blob/gh-pages/load_data.js
_demo_: http://chb.github.com/fhir_cardiac_risk?personId=1
